common/fw-update-helper.cpp

```diff
@@ -194,10 +194,61 @@ namespace rs2
         return false;
     }
 
+    void firmware_update_manager_mipi::process_flow(std::function<void()> cleanup, invoker invoke)
+    {
+        if (!_is_signed)
+        {
+            LOG_INFO("Only Signed Firmware can be burnt on MIPI device");
+            return;
+        }
+        auto dev_updatable = _dev.as<updatable>();
+        if(!(dev_updatable && dev_updatable.check_firmware_compatibility(_fw)))
+        {
+            fail("Firmware Update failed - fw version must be newer than version 5.13.1.1");
+            return;
+        }
+
+        // Enter DFU mode
+        auto device_debug = _dev.as<rs2::debug_protocol>();
+        uint32_t dfu_opcode = 0x1e;
+        device_debug.build_command(dfu_opcode, 1);
+
+        _progress = 30;
+        // Grant permissions for writing
+        // skipped for now - must be done in sudo
+        //chmod("/dev/d4xx-dfu504", __S_IREAD|__S_IWRITE);
+
+        // Write signed firmware to appropriate file descritptor
+        std::ofstream fw_path_in_device("/dev/d4xx-dfu504", std::ios::binary);
```

Comment:
Is this should always be hard coded? I am unfamiliar with how it works..

Reply:
For now, it will be hardcoded - in further PR about fw update (to be committed soon), I will check if it can be done in other ways